package.json

```diff
@@ -48,6 +59,8 @@
   },
   "devDependencies": {
     "ava": "^0.14.0",
+    "babel": "^5.8.38",
+    "nyc": "^6.4.4",
     "xo": "^0.13.0"
   }
 }
```

Comment:
Can you commit the changes in package.json and .gitignore directly? They can be useful for existing code and other PRs too. Then just rebase this PR.